src/tlo/methods/cervical_cancer.py

```diff
@@ -0,0 +1,1515 @@
+                and df.at[person_id, 'ce_date_treatment'] > (self.sim.date - pd.DateOffset(days=days_threshold_1825)):
+                hs.schedule_hsi_event(
+                    hsi_event=HSI_CervicalCancer_PostTreatmentCheck(
+                    module=self.module,
+                    person_id=person_id
+                    ),
+                    topen=self.sim.date + DateOffset(months=12),
+                    tclose=None,
+                    priority=0
+                )
+
+class HSI_CervicalCancer_PalliativeCare(HSI_Event, IndividualScopeEventMixin):
+    """
+    This is the event for palliative care. It does not affect the patients progress but does affect the disability
+     weight and takes resources from the healthsystem.
+    This event is scheduled by either:
+    * HSI_CervicalCancer_Biopsy following a diagnosis of cervical Cancer at stage4.
+    * HSI_CervicalCancer_PostTreatmentCheck following progression to stage4 during treatment.
+    * Itself for the continuance of care.
+    It is only for persons with a cancer in stage4.
+    """
+
+    def __init__(self, module, person_id):
+        super().__init__(module, person_id=person_id)
+
+        self.TREATMENT_ID = "CervicalCancer_PalliativeCare"
+        self.EXPECTED_APPT_FOOTPRINT = self.make_appt_footprint({})
+        self.ACCEPTED_FACILITY_LEVEL = '2'
+        self.BEDDAYS_FOOTPRINT = self.make_beddays_footprint({'general_bed': 15})
+
+    def apply(self, person_id, squeeze_factor):
+        df = self.sim.population.props
+        hs = self.sim.modules["HealthSystem"]
+
+        # todo: request consumables needed for this
+
+        if not df.at[person_id, 'is_alive']:
+            return hs.get_blank_appt_footprint()
+
+        # Check that the person is in stage4
+        assert df.at[person_id, "ce_hpv_cc_status"] == 'stage4'
+
+        # Record the start of palliative care if this is first appointment
+        if pd.isnull(df.at[person_id, "ce_date_palliative_care"]):
+            df.at[person_id, "ce_date_palliative_care"] = self.sim.date
+
+        # Schedule another instance of the event for one month
+        hs.schedule_hsi_event(
+            hsi_event=HSI_CervicalCancer_PalliativeCare(
+                module=self.module,
+                person_id=person_id
+            ),
+            topen=self.sim.date + DateOffset(months=1),
+            tclose=None,
+            priority=0
+        )
+
+
+# ---------------------------------------------------------------------------------------------------------
+#   LOGGING EVENTS
+# ---------------------------------------------------------------------------------------------------------
+
+class CervicalCancerLoggingEvent(RegularEvent, PopulationScopeEventMixin):
+    """The only logging event for this module"""
+
+    def __init__(self, module):
+        """schedule logging to repeat every 1 month
+        """
+        self.repeat = 30
+        super().__init__(module, frequency=DateOffset(days=self.repeat))
```

Comment:
how about changing days to months i.e. 
```suggestion
        self.repeat = 1
        super().__init__(module, frequency=DateOffset(months=self.repeat))
```

Reply:
good call. II think using months makes most sense.

Reply:
I think I wanted all periods the same length for comparability.